### ISSUE TITLE
View Editor morph not initializing or updating properly

### ISSUE THREAD

**Issue by rickmcgeer** [engageLively/galyleo-dashboard#95]
When initialized, the state of all checkboxes is selected, and when clicked the checkbox remains solid.  See this screenshot:
![view-editor-bug](https://user-images.githubusercontent.com/656259/193156971-67f7f8ef-fb75-4e79-b8e3-8902e01344cc.png)
The relevant code is in `SelectableEntryModel` in:
 `$world.execCommand("open browser", {moduleName: "studio/shared.cp.js", packageName: "galyleo-dashboard", codeEntity: [{"name":"SelectableEntryModel","type":"class-decl"},{"name":"toggleSelect","type":"class-instance-method"}]});`

A few things:

1. It appears to be unselected appropriately
2. When I directly execute `this.checkbox.master = this.isSelected ? CheckboxChecked : CheckboxUnchecked; `  it unchecks
3. When I then subsequently click to select, it checks
4. But then when I click again it doesn't uncheck
5. This is true even when I add the line `this.checkbox.master = this.isSelected ? CheckboxChecked : CheckboxUnchecked; ` to `toggleSelect`

**Comment by rickmcgeer** [comment]
I've also confirmed that the lists are being initialized properly, so the issue is with the Checkbox morph -- it simply isn't clearing when not selected

**Comment by merryman** [comment]
Yes, this appears to be a bug with nested master components. I will prepare a PR on `lively.next` ASAP.
